concordia/static/js/action-app/index.js

```diff
@@ -792,6 +805,28 @@ export class ActionApp {
         return sortBy(assetList, keyFromAsset);
     }
 
+    getCampaignOrTopic() {
+        let campaignOrTopic = 'campaign';
+
+        if (
+            this.campaignSelect.options[this.campaignSelect.selectedIndex]
+                .parentElement.label == 'Topics'
```

Comment:
> Reviewing this is making me think that we should rename the campaign selector to a less-specific name now that it's doing double-duty but am not coming up with non-awkward generic language. Maybe something like “containerFilter”?

I hear you on this, but I think we should wait until we see what happens next and whether we're going to refactor Campaigns/Topics so that the whole design is cleaner.